cvat/apps/dataset_manager/bindings.py

```diff
@@ -2151,11 +2154,67 @@ def reduce_fn(acc, v):
                 raise CvatImportError("Image {}: can't import annotation "
                     "#{} ({}): {}".format(item.id, idx, ann.type.name, e)) from e
 
-    for track in tracks.values():
+    finalize_tracks = {}
+    stop_frame = int(instance_data.meta[instance_data.META_FIELD]['stop_frame'])
+    for track_id, track in tracks.items():
+        finalize_tracks[track_id] = {
+            'label': track['label'],
+            'group': track['group'],
+            'source': track['source'],
+            'shapes': [],
+            'elements':{},
+        }
```

Comment:
I'm still don't see why this variable is needed, as you could just do everything with the current `track` in the loop, and maybe using a new `shapes` and `elements` variables.